### PR TITLE
Fix sub-modules when custom directories are used

### PR DESCRIPTION
**Pull request by parkerholladay**
Update the `git lg` alias to work on windows machines.

### CODE REVIEW

File: src/common/services/git.js [hunk 1]
```diff
@@ -69,26 +69,34 @@ function addPostCommitFiles(destination) {
...
+  return statuses.map(s => s.trim().split(' ')[1])
+}
+
+async function addPostCommitFilesToSubModules(repoPath) {
+  const submodulesPath = path.join(repoPath, '.git', 'modules')
+
+  if (fs.existsSync(submodulesPath)) {
+    const submodules = await getSubmodulesForRepo(repoPath)
+
+    submodules.map(modulePath => {
```
Comment (jimboslicethat) [inline src/common/services/git.js]: Nitpicky thing, but if we aren't using the value of `.map` we could probably just use `forEach`.

File: src/common/services/__specs__/git.spec.js [hunk 2]
```diff
@@ -208,73 +216,65 @@ describe('services/git', () => {
...
         let submodule1GitHooksPath
         let submodule2GitHooksPath
+        let submodule3GitHooksPath
 
         beforeEach(() => {
           submoduleExists = true
-          submoduleDirs = ['mod1', 'mod2']
+          submoduleDirs = ['mod1', 'subdir/mod2', 'subdir/mod3']
+          submoduleStatus = submoduleDirs
+            .map((dir, i) => `${i % 2 === 0 ? '+' : ' '}rando-commit-hash ${dir} (current/branch)`)
```
Comment (jimboslicethat) [inline src/common/services/__specs__/git.spec.js]: Not quite sure I fully understand the `i % 2` step.

Comment (parkerholladay) [reply src/common/services/__specs__/git.spec.js]: That adds a `+` sign to the front of even submodules. That is so we can test that the parsing works for both submodules that have been initialized (preceding `+`) and those that haven't.

File: src/common/services/git.js (hunk 1 above)
Comment (parkerholladay) [reply src/common/services/git.js]: Sounds good to me.

**Pull request merged**